crates/bevy_app/src/plugin_group.rs

```diff
@@ -78,6 +86,7 @@ impl PluginGroupBuilder {
         self
     }
 
+    /// Enables a [`Plugin`]
```

Comment:
Hmm. Perhaps it's useful to note that all plugins with the plugin group are enabled by default here, and that this is just to reverse manual disabling.